yark/archiver/config.py

```diff
@@ -110,3 +119,13 @@ def settings_md(self) -> YtDlpSettings:
 
         # Return
         return settings
+
+    def browser_url(self, archive_name: Optional[str]) -> str:
```

Comment:
Small one-line docstring here to explain what this with some comments below on main blocks

Reply:
Thanks :)